roles/collect_modified_images/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: "Create empty list for images"
+  set_fact:
+    pulled_images: []
+
+- name: "Generate events.json"
+  shell: "{% raw %}oc get events --field-selector reason==Pulling -o go-template='{{range .items}}{{.lastTimestamp}},{{.message}}{{\"\\n\"}}{{end}}' --sort-by={.lastTimestamp}{% endraw %}"
+  register: events
+
+- name: "get list of images in events.json"
+  include_tasks: parse_image_names.yml
+  with_items:
+    - "{{ events.stdout_lines }}"
+
+- name: "Print images"
+  debug: 
+  msg: "pulled images: {{ pulled_images }}"
+
+- name: "Save images to file"
+  local_action:
+    module: copy
+    content: "{{ pulled_images | join('\n') }}"
```

Comment:
I would prefer some structured text as an output instead of single string which has to be parsed

Reply:
Each image is listed on a new line, I thought that would be the most straightforward to parse.  If not, I can use some other delineation for the images such as a comma

Reply:
It is much reliable to delegate parsing to standard parsers for structural data.

Reply:
Alright, I can write it to a json file